.github/workflows/build.yml

```diff
@@ -0,0 +1,103 @@
+name: Build wheels
+
+on:
+  push:
+    branches: [master]
```

Comment:
I recommend removing this branch restriction, so contributors can test their feature branches before opening PRs.

Relatedly, do you have an example build?